src/ResourceManager/Network/ChangeLog.md

```diff
@@ -4,6 +4,7 @@
     Changes for the current release should go under the section titled "Current Release", and should adhere to the following format:
 
     ## Current Release
+    * Add DisableBgpRoutePropagation property in RouteTable and effectiveRoute.
```

Comment:
This should be added to the Current Release header (this is currently in the comments above and will not be included).

Reply:
Sure